app/src/main/java/com/example/farmeraid/data/model/FridgeModel.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.farmeraid.data.model
+
+import com.example.farmeraid.data.QuotasRepository
+
+class FridgeModel {
+    data class Fridge(
+        val fridgeName: String,
+        val location: String,
+        val items: List<QuotasRepository.ProduceQuota>
```

Comment:
I would suggest making this a new data class called ProduceFridge that has the following structure:
- produceName String
- prodouceDonateAmount: Int

The quota one also has saleAmount which won't be necessary for this use case.